README.md

```diff
@@ -1,4 +1,4 @@
-# dart2_fix
+# dart2_fix [![Build Status](https://travis-ci.org/dart-lang/dart2_fix.svg?branch=master)](https://travis-ci.org/dart-lang/dart2_fix)
```

Comment:
Don't need a title at the top. In every place where it's displayed, there is sufficient context.

Reply:
done